meshtastic/tests/test_api_baseline_comparison.py

```
@@ -0,0 +1,690 @@
+    n_args = len(func_node.args.args)
+    n_defaults = len(defaults)
+    for i, default_node in enumerate(defaults):
+        arg_idx = n_args - n_defaults + i
+        default_str = ast.unparse(default_node)
+        params[arg_idx] += f"={default_str}"
+    if func_node.args.vararg:
+        params.append(f"*{func_node.args.vararg.arg}")
+    if func_node.args.kwarg:
+        params.append(f"**{func_node.args.kwarg.arg}")
+    result = f"({', '.join(params)})"
+    return result
```

Ruff (0.15.7)

[warning] Unnecessary assignment to `result` before `return` statement

Remove unnecessary assignment

(RET504)